src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/SurveySvcRestClientImpl.java

```diff
@@ -175,7 +175,6 @@ public SurveyDTO findSurveyByRef(final String surveyRef) {
       if (e.getStatusCode() == HttpStatus.NOT_FOUND) {
         return null;
       }
-      log.error("Client error with status code = {}", e.getStatusCode(), e);
```

Comment:
why have we removed this?

Reply:
error gets thrown after anyway? Not sure what value this log adds